lib/Runtime/HostManager/HostManager.cpp

```diff
@@ -344,37 +383,42 @@ HostManager::runNetwork(llvm::StringRef networkName,
         std::move(context));
     return currentRun;
   }
-
+  // Setup the request
+  InferRequest queuedRequest;
```

Comment:
Why not have a proper constructor for `InferRequest`, instead of initializing fields one-by-one?